src/blockchain/esplora/ureq.rs

```diff
@@ -377,3 +377,11 @@ impl ConfigurableBlockchain for EsploraBlockchain {
         Ok(EsploraBlockchain::new(config.base_url.as_str(), config.stop_gap).with_agent(agent))
     }
 }
+
+#[cfg(test)]
+#[cfg(feature = "test-esplora")]
+crate::bdk_blockchain_tests! {
```

Comment:
I think you meant to move this to esplora/mod.rs as in your original #415 (and remove from esplora/reqwest.rs).

Reply:
If we want to be able to test both eslpora/reqwest _and_ esplora/ureq we need this to be in the ureq/reqwest submodules. I originally only put it in `mod.rs` thinking testing both was unnecessary, in hindsight this was wrong. With this change testing can be done by enabling feature `test-esplora` as well as the feature to enable the backend we want to test (e.g. `use-esplora-reqwest`).

Reply:
I locally tried moving this chunk to esplora/mod.rs and removing from `reqwest.rs` and `ureq.rs` and tests run fine with these commands:
```
cargo test --no-default-features --features test-esplora,use-esplora-ureq esplora::bdk_blockchain_tests
cargo test --no-default-features --features test-esplora,use-esplora-reqwest esplora::bdk_blockchain_tests
```